src/backend/config/KsnipConfig.cpp

```diff
@@ -327,6 +327,19 @@ void KsnipConfig::setTrayIconNotificationsEnabled(bool enabled)
 	saveValue(KsnipConfigOptions::trayIconNotificationsEnabledString(), enabled);
 }
 
+bool KsnipConfig::platformSpecificNotificationServiceEnabled() const
+{
+	return loadValue(KsnipConfigOptions::platformSpecificNotificationServiceEnabledString(), false).toBool();
```

Comment:
Not sure about the default value.

Reply:
Good question, I think I would go for true if it provides more functionality then what we have now.